src/csharp/Intel/Iced/Intel/Assembler/Assembler.cs

```diff
@@ -302,6 +302,73 @@ public sealed partial class Assembler {
 			AddInstruction(Instruction.CreateBranch(Bitness >= 32 ? Code.Jmp_ptr1632 : Code.Jmp_ptr1616, basePtr, address));
 		}
 		
+		/// <summary>
+		/// Generates multibyte NOP instructions
+		/// </summary>
+		/// <param name="amount">Amount of NOPs</param>
+		public void nop(int amount) {
+			if (amount <= 0)
+				throw new ArgumentOutOfRangeException($"{nameof(amount)} must be within 1-{int.MaxValue}");
```

Comment:
`throw new ArgumentOutOfRangeException(nameof(amount))` since that argument is the argument name, not a message.

Reply:
We should also allow 0 bytes so the caller doesn't have to check if it's > 0. If it's zero, just return.

Reply:
Changed